### ISSUE TITLE
Can you show how you manage your dotfiles?

### ISSUE THREAD

**Issue by supermario9590** [daviwil/emacs-from-scratch#8]
Your dotfiles repo README is not detailed enough for me, I would like to know how you set up gnu stow and your system to manage dotfiles.

**Comment by daviwil** [comment]
Yep!  I've been asked a few times about that so I'll do a video about it at some point :)
